### ISSUE TITLE
ImportError: No module named gpg

### ISSUE THREAD

**Issue by Futureknows** [micahflee/torbrowser-launcher#381]
TB on Fedora f29 Cinnamon is giving me the following error:

`$ /usr/bin/torbrowser-launcher
Traceback (most recent call last):
  File "/usr/bin/torbrowser-launcher", line 29, in <module>
    import torbrowser_launcher
  File "/usr/lib/python2.7/site-packages/torbrowser_launcher/__init__.py", line 36, in <module>
    from .common import Common, SHARE
  File "/usr/lib/python2.7/site-packages/torbrowser_launcher/common.py", line 38, in <module>
    import gpg
ImportError: No module named gpg
`

I've tried rebuilding from git, reinstalled all dependencies, dnf refresh upgrade.

**Comment by strasharo** [comment]
sudo dnf install python2-gpg

**Comment by strasharo** [comment]
Also got this:
torbrowser-launcher 
Traceback (most recent call last):
  File "/usr/bin/torbrowser-launcher", line 29, in <module>
    import torbrowser_launcher
  File "/usr/lib/python2.7/site-packages/torbrowser_launcher/__init__.py", line 34, in <module>
    from PyQt5 import QtCore, QtWidgets
ImportError: No module named PyQt5

Resolved it by installing this:
sudo dnf install python2-qt5-base

For some reason both gpg and python2-qt5-base are not listed as a dependency for the package:

rpm -qR torbrowser-launcher 
/usr/bin/python2
gnupg
pygtk2
pyliblzma
python(abi) = 2.7
python2
python2-psutil
python2-service-identity
python2-twisted
python2-txsocksx
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PartialHardlinkSets) <= 4.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1

**Comment by Futureknows** [comment]
Thanks, I installed both python2-gpg and python2-qt5-base.  Now I get: 

```
/usr/bin/torbrowser-launcher 
Traceback (most recent call last):
  File "/usr/bin/torbrowser-launcher", line 29, in <module>
    import torbrowser_launcher
  File "/usr/lib/python2.7/site-packages/torbrowser_launcher/__init__.py", line 38, in <module>
    from .launcher import Launcher
  File "/usr/lib/python2.7/site-packages/torbrowser_launcher/launcher.py", line 36, in <module>
    import requests
ImportError: No module named requests
```

**Comment by strasharo** [comment]
Please, keep the issue open, guess the devs will have to sort the dependency list for the new package in order to prevent these issues.

**Comment by SkewedZeppelin** [comment]
There is a report here https://bugzilla.redhat.com/show_bug.cgi?id=1662497
The package is an orphan, but someone updated it. Changes here https://src.fedoraproject.org/rpms/torbrowser-launcher/commits/master
Not sure how it got pushed out when it is missing so many deps.

It probably needs to be overhauled anyway, since 0.3.0 changed from python 2 to 3.

**Comment by SkewedZeppelin** [comment]
This can be closed now: 0.3.1-3 is now in F29 updates (and F28 testing) and installs the needed dependencies.
